managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/OpAddEntry.java

```diff
@@ -97,14 +97,12 @@ public void initiate() {
         ByteBuf duplicateBuffer = data.retainedDuplicate();
 
         // internally asyncAddEntry() will take the ownership of the buffer and release it at the end
-        lastInitTime = System.nanoTime();
+        this.ctxForCallbackComplete = ctx;
```

Comment:
nit: Remove `this.`

Reply:
umm.. actually ctx text is part of multiple variable and keeping `this` here makes more readable.